app/controllers/insured/consumer_roles_controller.rb

```diff
@@ -249,7 +249,7 @@ def update
   end
 
   def ridp_agreement
-    set_current_person
+    set_current_person unless current_user&.has_hbx_staff_role?
     consumer = @person.consumer_role
```

Comment:
What will `@person` be if we're not running `set_current_person`?

Reply:
https://github.com/ideacrew/enroll/blob/trunk/app/views/insured/consumer_roles/ridp_agreement.html.erb Don't think its a big deal cuz RIDP agreement is just some text without variables but we could revert it if necessary